view/adminhtml/web/tailwind/tailwind.config.js

```diff
@@ -1,5 +1,12 @@
 const colors = require("tailwindcss/colors");
-
+const magento = {
+    header: '#5f564f',
+    color: '#514943',
+    background: {
+        DEFAULT: '#f5f5f5',
+        hover: '#dbdbdb'
+    }
+}
```

Comment:
I'm fine with this, but like I said in slack, I don't know what I'm doing in regards to tailwind.
For now, let's keep it, and after the Hyvä frontend theme is released, we can ask Willem to have a look if he thinks something can be improved.